### PR TITLE
[sc-93181] Publish library

### PR DESCRIPTION
**Pull request by jess-ingraham**
Publishing for Swift Package Manager is actually pretty easy. We just need to tag the newest version! This is adding the initial files and package data

### CODE REVIEW

File: Sources/iosDesignTokens/todaytix/Tokens.swift [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+    public static let darkText04 = UIColor(red: 1.000, green: 1.000, blue: 1.000, alpha: 1)
+    public static let darkText05 = UIColor(red: 0.588, green: 0.588, blue: 0.624, alpha: 1)
+    public static let darkTextDiscount = UIColor(red: 1.000, green: 0.059, blue: 0.341, alpha: 1)
+    public static let darkTextError = UIColor(red: 1.000, green: 0.071, blue: 0.149, alpha: 1)
+    public static let darkUiUi01 = UIColor(red: 0.090, green: 0.086, blue: 0.122, alpha: 1)
+    public static let darkUiUi02 = UIColor(red: 0.173, green: 0.173, blue: 0.247, alpha: 1)
+    public static let darkUiUi03 = UIColor(red: 0.392, green: 0.388, blue: 0.455, alpha: 1)
+    public static let darkUiUi04 = UIColor(red: 0.588, green: 0.588, blue: 0.624, alpha: 1)
+    public static let darkUiUi05 = UIColor(red: 1.000, green: 1.000, blue: 1.000, alpha: 1)
+    public static let fontFamiliesBody = "Aktiv Grotesk App"
```
Comment (AndresLeonardoMartinez) [inline Sources/iosDesignTokens/todaytix/Tokens.swift:163]: Do we need to add the fonts as well or just define the names?

**Review by AndresLeonardoMartinez — APPROVED** [review]
🎉

File: Package.swift [hunk 2]
```diff
@@ -5,24 +5,21 @@ import PackageDescription
...
         // Targets can depend on other targets in this package, and on products in packages this package depends on.
         .target(
             name: "iosDesignTokens",
             dependencies: []),
-        .testTarget(
-            name: "iosDesignTokensTests",
-            dependencies: ["iosDesignTokens"]),
+//        .testTarget(
+//            name: "iosDesignTokensTests",
+//            dependencies: ["iosDesignTokens"]),
```
Comment (flitterio) [inline Package.swift:23]: do we still want this here?

Comment (Jeremias-Pellegrino) [inline Package.swift:23]: These left as comments?

File: Sources/iosDesignTokens/todaytix/GradientTokens.swift [hunk 3]
```diff
@@ -0,0 +1,149 @@
+
+// Do not edit directly
+// Generated on Thu, 10 Nov 2022 19:19:40 GMT
+
+
+import UIKit
+
+public class GradientTokens {
+	public var lightInteractiveInteractive01: CAGradientLayer {
```
Comment (flitterio) [inline Sources/iosDesignTokens/todaytix/GradientTokens.swift:9]: I am not sure what this is based on the name

File: Sources/iosDesignTokens/todaytix/Tokens.swift [hunk 4]
```diff
@@ -0,0 +1,282 @@
...
+public enum Tokens {
+    public static let borderRadiusLg = CGFloat(8)
+    public static let borderRadiusSm = CGFloat(4)
+    public static let borderRadiusXl = CGFloat(16)
+    public static let borderWidthLg = CGFloat(8)
+    public static let borderWidthMd = CGFloat(4)
+    public static let borderWidthNone = CGFloat(0)
+    public static let borderWidthSm = CGFloat(2)
+    public static let borderWidthXs = CGFloat(1)
+    public static let coreBasic0 = UIColor(red: 1.000, green: 1.000, blue: 1.000, alpha: 1)
```
Comment (Jeremias-Pellegrino) [inline Sources/iosDesignTokens/todaytix/Tokens.swift:21]: 3 decimals of precision? Whoa D:

File: Sources/iosDesignTokens/todaytix/GradientTokens.swift [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+			gradientLayer.startPoint = CGPoint(x: 0.5, y: 0.05)
+			gradientLayer.endPoint = CGPoint(x: 0.5, y: 0.95)
+			return gradientLayer
+		}
+	}
+
+	public var coreGradientRaspberry: CAGradientLayer {
+		get {
+			let gradientLayer = CAGradientLayer()
+			gradientLayer.colors = [UIColor(red: 0.502, green: 0.024, blue: 0.220, alpha:1), UIColor(red: 0.882, green: 0.090, blue: 0.408, alpha:1)]
```
Comment (flitterio) [inline Sources/iosDesignTokens/todaytix/GradientTokens.swift:62]: are these all just auto generated? I feel like we don't have these anywhere

**Review by flitterio — APPROVED** [review]

**Review by Jeremias-Pellegrino — APPROVED** [review]
Just saw commented lines, rest looks good! 🚀

File: Sources/iosDesignTokens/todaytix/GradientTokens.swift (hunk 5 above)
Comment (Jeremias-Pellegrino) [reply Sources/iosDesignTokens/todaytix/GradientTokens.swift:62]: @flitterio Isn't this the gradients used for `get tickets` buttons and similar?

File: Sources/iosDesignTokens/todaytix/Tokens.swift (hunk 1 above)
Comment (jess-ingraham) [reply Sources/iosDesignTokens/todaytix/Tokens.swift:163]: just defining the names for now - still sort of deciding how to do typography. This is all still a work in progress

File: Sources/iosDesignTokens/todaytix/GradientTokens.swift (hunk 3 above)
Comment (jess-ingraham) [reply Sources/iosDesignTokens/todaytix/GradientTokens.swift:9]: it's based on the token path. When we generate the token files with amazon style dictionary, this is the name it gives us. Sean is working on naming so this isn't necessarily final or anything - I just needed something to test with.

File: Sources/iosDesignTokens/todaytix/GradientTokens.swift (hunk 5 above)
Comment (jess-ingraham) [reply Sources/iosDesignTokens/todaytix/GradientTokens.swift:62]: yeah, we use gradients for the primary action button and I needed to see if gradients could work in this format

File: Sources/iosDesignTokens/todaytix/Tokens.swift (hunk 4 above)
Comment (jess-ingraham) [reply Sources/iosDesignTokens/todaytix/Tokens.swift:21]: that's what was generated. Is that not ok? I think I can change that in the script but I'm not 100% sure

File: Package.swift (hunk 2 above)
Comment (jess-ingraham) [reply Package.swift:23]: I kept it in case we ever wanted tests but I can remove it

Comment (jess-ingraham) [reply Package.swift:23]: I kept it in case we ever wanted tests but I can remove it

**Pull request merged**